kolibri/plugins/learn/assets/src/views/classes/Hint.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+
+  <div class="hint-btn-container">
+    <KButton
+      class="hint-btn"
+      appearance="basic-link"
+      :text="$tr('hint', { hintsLeft: availableHints })"
+      :primary="false"
+      @click="takeHint"
+    />
+    <CoreInfoIcon
+      class="info-icon"
+      tooltipPosition="bottom right"
+      :iconAriaLabel="$tr('hintExplanation')"
+      :tooltipText="$tr('hintExplanation')"
+    />
+  </div>
+
+</template>
+
+
+<script>
+
+  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
```

Comment:
I'm not seeing where `responsiveWindowMixin` is being used here (or in a few other components). I believe that, unless you're going to directly use one of the helpers defined in [KResponsiveWindowMixin](https://github.com/learningequality/kolibri-design-system/blob/main/lib/KResponsiveWindowMixin.js) there's no need to include it.